### PR TITLE
Add XLink Kai.app v7.4.35

### PR DESCRIPTION
**Pull request by sgtlegend**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/xlink-kai.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  version '7.4.35'
+  sha256 'ebc988dd6491439b897f8f6e51f870a919f0d416a7d6096a62441c8e7bf05d05'
+
+  # github.com/Team-XLink/releases was verified as official when first introduced to the cask
+  url "https://github.com/Team-XLink/releases/releases/download/v#{version}/XLink.Kai.app.zip"
+  appcast 'https://github.com/Team-XLink/releases/releases.atom'
+  name 'XLink Kai'
+  homepage 'https://www.teamxlink.co.uk/'
+
+  depends_on arch: :intel
```
Comment (vitorgalvao) [inline Casks/xlink-kai.rb]: Remove this line. Homebrew itself doesn’t run on other architectures.

File: Casks/xlink-kai.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cask 'xlink-kai' do
+  version '7.4.35'
+  sha256 'ebc988dd6491439b897f8f6e51f870a919f0d416a7d6096a62441c8e7bf05d05'
+
+  # github.com/Team-XLink/releases was verified as official when first introduced to the cask
+  url "https://github.com/Team-XLink/releases/releases/download/v#{version}/XLink.Kai.app.zip"
+  appcast 'https://github.com/Team-XLink/releases/releases.atom'
```
Comment (vitorgalvao) [inline Casks/xlink-kai.rb]: Where on the website do they link to github?

File: Casks/xlink-kai.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+cask 'xlink-kai' do
+  version '7.4.35'
+  sha256 'ebc988dd6491439b897f8f6e51f870a919f0d416a7d6096a62441c8e7bf05d05'
+
+  # github.com/Team-XLink/releases was verified as official when first introduced to the cask
+  url "https://github.com/Team-XLink/releases/releases/download/v#{version}/XLink.Kai.app.zip"
```
Comment (vitorgalvao) [inline Casks/xlink-kai.rb]: Use the link on the page.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]
In the future, please don’t untick the box that lets maintainers edit the PR. It saves time for everyone.

File: Casks/xlink-kai.rb (hunk 2 above)
Comment (sgtlegend) [reply Casks/xlink-kai.rb]: I have added a link on our downloads page.

https://www.teamxlink.co.uk/?go=download

**Comment by sgtlegend** [comment]
> In the future, please don’t untick the box that lets maintainers edit the PR. It saves time for everyone.

I didn't change any settings when I opened it, I just edited the tasks and submitted it.

File: Casks/xlink-kai.rb (hunk 3 above)
Comment (sgtlegend) [reply Casks/xlink-kai.rb]: Which link are you referring to?

**Pull request merged**